### DISCUSSION TITLE
Stop training model when accuracy reached greater than 0.99

### DISCUSSION THREAD

**Discussion by jeremiaww** [teamalgoritma/community#136] [Q&A] [answered]
Apakah bisa melakukan stop training model (stop epoch) ketika akurasi model nya sudah tercapai (melewati threshold), dengan menggunakan Keras di R.
Alasannya agar, ketika akurasi nya sudah tercapai tidak perlu lama menunggu model selesai training.

Untuk contoh di phyton yang saya coba cari seperti berikut:
https://stackoverflow.com/questions/67216419/stop-training-model-when-accuracy-reached-greater-than-0-99

Terima kasih.

**Comment by kevbow** [comment, marked as answer]
Halo @jeremiaww,

Bapak dapat menggunakan konsep callback seperti yang tercantum pada link [berikut ini](https://keras.rstudio.com/articles/training_callbacks.html).
Fungsi pengecekan akurasi dapat menggunakan custom callback dengan nilai-nilai yang disesuaikan dengan kebutuhan. Silakan melakukan eksplorasi mandiri terkait hal tersebut.

Good luck!

**Reply by jeremiaww** [reply]
Terima kasih mas atas sharingnya, akan dicoba nanti.
